### PR TITLE
Don't show jump to add comment if add comment form is missing

### LINKED ISSUE #3943: Don't show "jump to add comment" button if "add comment" form is missing
e.g. on a comment permalink page or if banned from a subreddit

### PR DESCRIPTION
**Pull request by preethamrn**
Fixes #3943

### CODE REVIEW

File: lib/modules/pageNavigator.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ function backToTop() {
 
 function backToNewCommentArea() {
 	const commentArea = document.querySelector('.commentarea > form.usertext textarea');
-	if (!commentArea) return;
+	if (!commentArea || !commentArea.offsetParent) return;
 	const $backToNewCommentArea = $(`<a class="pageNavigator res-icon" data-id="addComment" href="#comments" title="${i18n('pageNavToCommentTitle')}">&#xF003;</a>`);
```
Comment (andytuba) [inline lib/modules/pageNavigator.js]: Oh right, it's rendered so it can be cloned for replies to comments but hidden initially.

**Comment by erikdesjardins** [comment]
(integration test failure is unrelated)

**Comment by erikdesjardins** [comment]
Thanks!

**Comment by preethamrn** [comment]
No problem :)

**Pull request merged**